mappings/net/minecraft/util/registry/SimpleRegistry.mapping

```diff
@@ -5,4 +5,8 @@ CLASS net/minecraft/class_2370 net/minecraft/util/registry/SimpleRegistry
 	FIELD field_11110 indexedEntries Lnet/minecraft/class_3513;
 	FIELD field_11111 LOGGER Lorg/apache/logging/log4j/Logger;
 	FIELD field_25067 entriesByKey Lcom/google/common/collect/BiMap;
+	FIELD field_25489 registryKeys Ljava/util/Set;
 	METHOD method_10240 getRandom (Ljava/util/Random;)Ljava/lang/Object;
+	METHOD method_29722 getEntriesByKey ()Ljava/util/Set;
+	METHOD method_29723 containsRegistryKey (Lnet/minecraft/class_5321;)Z
+	METHOD method_29725 addRegistryKey (Lnet/minecraft/class_5321;)V
```

Comment:
This is actually marking certain keys as loaded from encoder than builtin from code. Addressed in my pr